sidebars.js

```diff
@@ -105,6 +105,22 @@ const sidebars = {
         },
       ],
     },
+    {
+      type: "category",
+      label: "Partner Showroom",
+      collapsible: true,
+      collapsed: true,
+      link: {
+        type: 'generated-index',
+        slug: '/partner-showroom',
+      },
+      items: [
+        {
+          type: "autogenerated",
+          dirName: "partners",
```

Comment:
We should stick with only one name instead of having "/partner-showroom" and "partners". 

Currently, when you click on Partner Showroom, the URL will be `/partner-showroom`. But after you click one of the pages, it will be `/partner/introduction`.

Reply:
good suggestion, made it consistent.